scripts/benchmarks/benches/compute.py

```diff
@@ -18,11 +18,13 @@ def __init__(self, directory):
         return
 
     def setup(self):
-        if self.built:
+        build_path = create_build_path(self.directory, 'compute-benchmarks-build')
+        self.bins = os.path.join(build_path, 'bin')
+
+        if self.built or not options.rebuild:
```

Comment:
why?
self.built means that this was already built initialized by the previous invocation of setup. What rebuilt is preventing is removing the build directory, but we still want to pull the changes, call cmake and see whether there aren't any updates.

Reply:
Ok, if it is crucial. The change made it possible to run the scripts on cluster nodes with no access to external servers...